plugins/woocommerce-admin/client/customize-store/intro/services.ts

```diff
@@ -8,6 +8,7 @@ export const fetchThemeCards = async () => {
 			description: 'The default theme for WordPress.',
 			image: 'https://i0.wp.com/s2.wp.com/wp-content/themes/pub/twentytwentyone/screenshot.png',
 			styleVariations: [],
+			isActive: true,
```

Comment:
this feels like a footgun that could come back to bite us, should we leave isActive out until we put in the logic to determine what theme is active?

Reply:
We will replace this entirely with an API call to fetch theme data, so I think it should be fine. 🙂